src/rock/models/feeds/resolver.js

```diff
@@ -43,6 +86,13 @@ export default {
         filterQueries.push(models.Like.getLikedContent(user._id, models.Node));
       }
 
+      //for news section in app -- all news
+      if (filters.includes("NEWS")) {
```

Comment:
is this needed? We don't send a "NEWS" filter? Just "news" included in the channel list